docs/opcodes/55/berlin.mdx

```diff
@@ -1,5 +1,11 @@
 ## Gas
 
+### Definitions:
+- **value**: Value from the stack input.
```

Comment:
Last nitpick: the rest of the doc does not use upper case after ':', can you change it to lower case?

Reply:
+1, I changed it

Reply:
changed it